Android/ThirdPartyAuth/app/src/main/java/com/bentley/sample/thirdpartyauth/LoginActivity.kt

```diff
@@ -20,13 +24,13 @@ import androidx.compose.ui.unit.sp
 import com.bentley.sample.thirdpartyauth.ui.theme.ThirdPartyAuthTheme
 
 class LoginActivity : ComponentActivity() {
-
+    
```

Comment:
You probably want to search the entire project for a regex of `[ ]+$` and delete any that are found.

Reply:
Keeping indents on empty lines is a project setting. I personally like it so I don't have to type spaces for all previous indents, but I can turn it off if it's unwanted

Reply:
removed